### DISCUSSION TITLE
How can I go to an address ?

### DISCUSSION THREAD

**Discussion by yeetdevDave** [eteran/edb-debugger#852] [Q&A] [answered]
![image](https://github.com/eteran/edb-debugger/assets/107414336/f1768316-fcb2-4816-acc9-e66c4880467f)
I wanna go to ECX's value address 
But I don't know how
![image](https://github.com/eteran/edb-debugger/assets/107414336/9455eb43-56a3-4823-be30-ae7327aef340)

**Comment by 10110111** [comment, marked as answer]
Right-click the register → _Follow in Dump_.
Or: click inside the _Dump_, <kbd>Ctrl</kbd>+<kbd>g</kbd>, type `rcx`, OK.
